models/edr/alerts/error_models.sql

```diff
@@ -0,0 +1,23 @@
+{{
+  config(
+    materialized = 'view',
+    bind=False
+  )
+}}
+
+select model_execution_id as alert_id,
+       unique_id,
+       generated_at as detected_at,
+       database_name,
+       materialization,
+       path,
+       original_path,
+       schema_name,
+       message,
+       owner as owners,
+       tags,
+       alias,
+       status,
+       full_refresh
+from {{ ref('model_run_results') }}
```

Comment:
Let's be explicit here and add 'elementary' to the ref? it's just that model_run_results is a name I think other artifact packages can use as well